CMakeLists.txt

```diff
@@ -50,6 +50,16 @@ else()
   message(STATUS "Configuring Shaderc to avoid building examples.")
 endif()
 
+option(SHADERC_SKIP_COPYRIGHT_CHECK "Skip copyright check" ${SHADERC_SKIP_COPYRIGHT_CHECK})
+if(NOT ${SHADERC_SKIP_COPYRIGHT_CHECK})
```

Comment:
Thanks for keeping the check in the default flow.
So I'm ok with doing this.

Another fix would be to update the filtering parameters in https://github.com/google/shaderc/blob/main/utils/add_copyright.py#L64

Reply:
Thanks for your time!

There doesn't seem to be an easy way that wouldn't modify the whole algorithm in a way that could have unwanted side effects, or am I missing something?  The current `find()` function cannot filter out particular filenames (it filters out directory names).  To block `.\build\include\glslang\build_info.h` from the check, I'd either have to add the whole `glslang` directory to the `skip_glob_list` parameter (which might be too much), or add a new parameter that allows removal of filenames that match a pattern (which seems to be overkill for a single file, but is feasible, and wouldn't likely have tremendous side effects).

I'm happy to re-implement with a filtering algorithm modification if you prefer.

Reply:
It should have filtered out `\build\` because that matches `build*`.  Is the offending file in `dbuild` ?

In any case, I'm still ok landing this, and will do so.

One other thing comes to mind, which is I usually configure builds to avoid writing into the same tree as the source tree.  But that's not always so easy to control with IDEs, so I won't be fussy about that.

Reply:
> It should have filtered out \build\ because that matches build*. Is the offending file in dbuild ?

Ah, yes, that's it - I missed that detail.  Yes, the failed copyright message was indeed coming from `dbuild`.  So it would have been easy to add 'dbuild*` to the list of ignored directories... I apologize for my errant analysis.

> In any case, I'm still ok landing this, and will do so.

Thank you!